source/index.rst

```diff
@@ -75,13 +75,15 @@ These are institutions who were early adopters or provided HPC resources for dev
   :caption: How-Tos
 
   how-tos/apps.rst
+  how-tos/app-development
+  how-tos/tutorials-interactive-apps
```

Comment:
I kind of like just having the tutorials under the How-Tos, since a tutorial is just showing you "how to do something" it seems like it makes sense to me. It also prevents that TOC from becoming cluttered with sections. I'd find it kind of odd to be a user and see a "how to" and then a "tutorial" section, it just seems semantically redundant.

I'm happy either way though, and regardless the passenger tutorial should also be moved either into the Tutorials if we go that route, or also under the "How Tos".

Reply:
Broke out into the Tutorials and How-Tos sections and put the passenger and interactive app tutorials under the former and the app development under the latter.